src/lib/helpers/logger.ts

```diff
@@ -0,0 +1,42 @@
+enum EVerbosity {
+    WARN = 0,
+    INFO,
+    DEBUG,
+    DIAGNOSTIC,
+}
+
+class Logger {
```

Comment:
If you intend to have production/test logs outside of local testing, please follow the dApp system https://github.com/aresrpg/aresrpg-dapp/blob/master/src/logger.js

Reply:
Fixed in:
- 8f61367107900179860cfca38dc7b773f64c8bd4 which matches the logs style with the dapp logs
- 4c9ac6892c02c6f36d3dabd2e303deb74fa0210a which lowers the default verbosity to only show warnings